lib/reek/configuration/configuration_converter.rb

```diff
@@ -80,6 +80,8 @@ def strings_to_regexes_for_detectors
                 to_regex item
               end
             end
+          rescue NoMethodError
+            warn_about_missing_configuration(detector) if detectors[detector].nil?
```

Comment:
I think it's better to detect that `detector[detector]` is `nil` at the beginning of this block and handle that specific case there.